cli/src/main/java/org/jboss/as/cli/impl/ReadlineConsole.java

```diff
@@ -556,7 +557,7 @@ private void printCollectedOutput() {
             }
             // '\R' will match any line break.
             // -1 to keep empty lines at the end of content.
```

Comment:
This is no more relevant, I think a comment to explain what happens with empty lines would help